src/modules/git_status.rs

```diff
@@ -244,6 +247,17 @@ fn is_untracked(status: Status) -> bool {
     status.is_wt_new()
 }
 
+fn stashed_count(repository: &mut Repository) -> usize {
+    let mut count = 0;
+    repository
+        .stash_foreach(|_, _, _| {
+            count += 1;
+            true
+        })
+        .unwrap();
```

Comment:
This `.unwrap()` could be a little dangerous here.
Ideally we would return an `Option` or `Result`, and early return the module as `None` if this is an `Error`.